frontend/src/console/components/ButtonStyles/ButtonStyles.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { InstanceSettingsModel } from 'console/models';
+import { ThemeSchema } from 'ocim-client';
+import { Col, Container, Row } from 'react-bootstrap';
+import { CollapseEditArea, ColorInputField } from 'ui/components';
+import { WrappedMessage } from 'utils/intl';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface ButtonStylesProp {
+  onChangeColor: Function;
+  buttonName: string;
+  themeData: ThemeSchema;
+  initialExpanded?: boolean;
+  loading: Array<keyof InstanceSettingsModel | 'deployment'>;
+}
+
+export const ButtonStyles: React.FC<ButtonStylesProp> = (
+  props: ButtonStylesProp
+) => {
+  const { themeData } = props;
+  const { loading } = props;
+
+  const styles = {
+    activeStyles: [
+      `btn${props.buttonName}Bg`,
+      `btn${props.buttonName}Color`,
+      `btn${props.buttonName}BorderColor`
+    ],
+    hoverStyles: [
+      `btn${props.buttonName}HoverBg`,
+      `btn${props.buttonName}HoverColor`,
+      `btn${props.buttonName}HoverBorderColor`
+    ]
+  };
+
+  return (
+    <div>
```

Comment:
Do we need this outer `div`? Maybe a fragment `<>` or `CollapseEditArea` as root element would be just fine.